app/client/src/tests/system/createCommunity.e2e.test.js

```diff
@@ -0,0 +1,43 @@
+import { UserActions } from "../functions/UserActions";
+import puppeteer from "puppeteer";
+
+
+jest.setTimeout(30000);
+
+const idir = process.env.IDIR 
+const password = process.env.PASSWORD
+
+describe("Given that user is on login page", () => {
+  let browser;
+  let page;
+  let user;
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch({
+      headless: false,
+      slowMo: 10,
```

Comment:
If you want, there are environment variables for headless and slowmo. That way it's not hard coded into any tests.

Reply:
ya i will do that. also don't work too hard. its your first day being sick.